### PR TITLE
2274 icon gatsby

### LINKED ISSUE #2274: Update new Icon docs (Icon Gallery) & Add Icon Docs to NDS
**User Story:**
As Spark, we want to update our new icon Docs with Icon Gallery info where needed before adding the new icon documentation to SB

**Notes:**
- New Docs can be found on sharepoint or at shorty/sparkcontent  

**AC:**
- updated docs where necessary with Icon Gallery Info
- I will see updated documentation displaying on the New Doc Site

### PR DESCRIPTION
**Pull request by RCopeland**
## What does this PR do?
A clear and concise description of what this PR does.

### Associated Issue 
Fixes #2274 

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [x] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [x] Mozilla Firefox (Mobile)
  - [x] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [x] Microsoft Edge
  - [x] Apple Safari
  - [x] Apple Safari (Mobile)

### CODE REVIEW

File: angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts [hunk 1]
```diff
@@ -15,7 +15,33 @@ export default {
...
+will need to include the Icon yourself. Visit our
+guide on Installing Spark Icons.
+
+#### Accessibility
+
+Icons can simply be decorative or intended to convey
+meaning. Each use-case has separate accessibility
+considerations.
+
+Decorative Icons are purely visual or used to reinforcing accompanying
```
Comment (guern1kn) [inline angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts]: This should probably be "reinforce"

**Review by guern1kn — CHANGES_REQUESTED** [review]
Update "reinforcing" to "reinforce" in 3 files

**Review by guern1kn — APPROVED** [review]

File: angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts [hunk 2]
```diff
@@ -15,7 +15,33 @@ export default {
...
   ],
   parameters: {
-    info: markdownDocumentationLinkBuilder('icon'),
+    info: `
+${markdownDocumentationLinkBuilder('icon')}
+
+The icons in Spark are the property of Quicken
+Loans and are not provided directly in our packages. You
+will need to include the Icon yourself. Visit our
+guide on Installing Spark Icons.
```
Comment (afebbraro) [inline angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts]: Do we have this guide available yet? If its not going to be available for launch we should hold off on mentioning it. If it is available, we should link it up here.

File: src/components/ComponentPreview.js [hunk 3]
```diff
@@ -25,9 +27,11 @@ class ComponentPreview extends React.Component {
         <div className="sprk-u-mbl">
           <IframeResizer
             style={{
-              minHeight: minHeight,
-              maxWidth: maxWidth
+              minHeight,
```
Comment (afebbraro) [inline src/components/ComponentPreview.js]: nice!

**Review by afebbraro — APPROVED** [review]
Approved, just would like that one comment I left about linking the icon install guide addressed. Woot!

File: angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts (hunk 2 above)
Comment (RCopeland) [reply angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts]: https://github.com/sparkdesignsystem/spark-design-system/issues/2588

**Pull request merged**